build-common/NHibernate.props

```diff
@@ -2,14 +2,11 @@
   <Import Condition="Exists('NHibernate.dev.props')" Project="NHibernate.dev.props"/>
 
   <PropertyGroup>
-    <VersionMajor Condition="'$(VersionMajor)' == ''">5</VersionMajor>
-    <VersionMinor Condition="'$(VersionMinor)' == ''">2</VersionMinor>
-    <VersionPatch Condition="'$(VersionPatch)' == ''">7</VersionPatch>
-    <VersionSuffix Condition="'$(VersionSuffix)' == ''"></VersionSuffix>
+    <VersionPrefix Condition="'$(VersionPrefix)' == ''">5.3.0</VersionPrefix>
```

Comment:
This would require to manually support assembly version pinning: it should be 5.3.0 for entire 5.3.x branch.

Reply:
Sorry - I don't get what it means? Could you please clarify? What would require it?

Reply:
Got it.. )

Reply:
Updated comment. Hopefully it is clearer now.

Reply:
Regex replace can handle it.

Reply:
https://xkcd.com/1171/

Reply:
I don't get hazzik point at all. And why overkill? It's single line change. With your suggested change we have to do additional modifications in common.xml to properly construct `${project.version}`. For what?

Reply:
regex has been removed